App/SB/components/ThreadDetailCard/index.tsx

```diff
@@ -182,7 +182,7 @@ const mapStateToProps = (state: RootState, ownProps: OwnProps): StateProps => {
   // Unsquares the images by maintaining the aspect ratio no matter device size
   const imageWidth = WIDTH
   const links = photo.files[0].links
-  const meta = links ? links['large'].meta : undefined
+  const meta = links ? links['thumb'].meta : undefined
```

Comment:
This is a bit of a hack now... switched all these metadata queries to `thumb` since shared and private threads both have a `thumb` link and for now so these screens can be used to display images from either type of thread. Improvements tracked in #764